documentation.html

```diff
@@ -465,9 +473,137 @@ <h4>Button variants</h4>
 <button class="btn btn--primary btn--round btn--outline">Primary</button>
 <button class="btn btn--primary btn--ghost">Primary</button>
     </code>
+</pre>
+                    </section>
+
+                    <!------------------------------------------------------- Image section ------------------------------------------------------->
+                    <section id="image">
+                        <h1>Images</h1>
+                        <p>Felix UI comes with few utility classes for images for different purpose.</p>
+                        <h4>Responsive image</h4>
+                        <p>Images in Felix UI are made responsive with <code>.img-fluid</code>. This applies <code>max-width: 100%;</code> and <code> height: auto;</code> to the image so that it scales with the parent width.</p>
+                        <div class="element_container">
+                            <img src="/images/forest.jpg" class="img-fluid" alt="" />
+                        </div>
+                        <pre>
+    <code class="html">
+<img src="/images/forest.jpg" class="img-fluid" alt="" />
+    </code>
+</pre>
+                        <h4>Rounded images</h4>
+                        <div class="element_container">
+                            <img src="/images/flower.jpg" class="img-fluid round" alt="" />
+                            <img src="/images/sp.jpg" class="img-fluid round" alt="" />
```

Comment:
Everything seems good, Just add values to the alt attributes.

Reply:
everything is fine :)